### ISSUE TITLE
How to adjust the enhance degree and contrast degree for over exposure dataset

### ISSUE THREAD

**Issue by lastbasket** [cuiziteng/Aleth-NeRF#13]
Hi, I am training on my own over-exposure dataset. I find that the enhance degree and contrast degree are difficult to adjust and will result in nan in terms of training loss and psnr.

**Comment by cuiziteng** [comment]
Hi, thanks for using our project ~

Due to I can't get your own data, so I guess that there maybe some problem occurred in the loss function part. Please check the value in the inverse tone mapping loss function, if this is problem, please change it to the original MSE loss function.

https://github.com/cuiziteng/Aleth-NeRF/blob/7e1ef4116552faedb1af3966fb41e79a61eb0f2a/src/model/aleth_nerf_exp/helper.py#L23

**Comment by lastbasket** [comment]
Thanks, I clip both x, y and the problem is resolved.
